browser-app/package.json

```diff
@@ -27,10 +27,19 @@
     "@theia/cli": "next"
   },
   "scripts": {
-    "prepare": "theia build --mode development",
-    "start": "theia start",
-    "test": "true",
-    "watch": "theia build --watch --mode development"
+    "prepare": "yarn run clean && yarn build",
+    "clean": "theia clean && rimraf errorShots",
+    "build": "theiaext compile && theia build --mode development",
+    "watch": "concurrently -n compile,bundle \"theiaext watch --preserveWatchOutput\" \"theia build --watch --mode development\"",
+    "start": "theia start --plugins=local-dir:../../plugins",
+    "start:debug": "yarn start --log-level=debug",
+    "test": "wdio wdio.conf.js",
+    "test-non-headless": "wdio wdio-non-headless.conf.js",
+    "coverage:compile": "yarn build --config coverage-webpack.config.js",
```

Comment:
I am not sure we need the code coverage part. AFAIK it's not being used in the main repo. Has it been tested here?